pythonScript.py

```diff
@@ -12,6 +12,16 @@
 import sys, os, random, time
 
 if __name__ == "__main__":
+    print "Script depreciated, use instead:"
```

Comment:
LOL, this script is here for the sole purpose of having a place to hack and try shit. I don't really care what goes in here.